admin_ui/src/components/RowFormSelect.vue

```diff
@@ -109,13 +113,8 @@ export default Vue.extend({
             return value
         },
         getKeySelectID(propertyTitle: string) {
-            return (
-                this.keySelectIDs[propertyTitle] || this.getValue(propertyTitle)
-            )
-        },
-        keySelectChanged(propertyTitle: string, value: number) {
-            console.log(`${propertyTitle} = ${value}`)
-            Vue.set(this.keySelectIDs, propertyTitle, value)
+            localStorage.setItem("rowID", this.getValue(propertyTitle))
+            return localStorage.getItem("rowID")
```

Comment:
This might not have worked because `Vue.set` was removed in Vue 3. If we replace it with `Reflect.set`, or just `this.keySelectIDs[propertyTitle] = value` does it work as it was before?

Reply:
I’ve tried both before, but neither way works. I think localStorage is the good way to store FK `rowID` via browser tabs, and since `rowID` is not sensitive information, I think that's fine.